src/components/topRail/TopRail.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div class="top-rail-container">
+    <ValidateIssue
+      v-show="isOpenIssue"
+      :number-of-errors="numberOfErrors"
+      @openPanel="handleOpenPanel"
+    />
+
+    <ValidateButton @openIssue="handleOpenIssue" />
+
+    <ValidatePanel
+      v-show="isOpenIssue && isOpenPanel"
+      :error-list="errorList"
+      :warnings="warnings"
+    />
+  </div>
+</template>
+
+<script>
+import store from '@/store';
+import { ValidateButton, ValidateIssue, ValidatePanel } from '@/components/topRail/validateControl';
+
+export default {
+  components: {
+    ValidateButton,
+    ValidateIssue,
+    ValidatePanel,
+  },
+  props: ['validationErrors', 'warnings'],
```

Comment:
please define type of the props.
example 
```
props: {
    title: String,
    likes: Number
  }
```

Reply:
It will be fixed